app/views/users/circuitverse/edit.html.erb

```diff
@@ -120,10 +114,16 @@
       class: 'autocomplete-list-item',
     }
   });
-
   $(SELECTOR).on('selection', function(event) {
     const feedback = event.detail;
     const selection = feedback.selection.value[feedback.selection.key];
     autoCompleteJS.input.value = selection;
   });
+  document.querySelector('#remove_pic').addEventListener('change', function() {
+    if (this.checked) {
+      document.querySelector('#imgedit').src = '<%= image_path('thumb/Default.jpg') %>';
+    } else {
+      document.querySelector('#imgedit').src = '<%= @profile.profile_picture %>';
```

Comment:
@shauryag2002 can you give the logic behind the check ?

Because , expected behaviour would be just to remove previous image / uploaded image and set default one only 
right ?

Reply:
Ok , so i should only set the image to default one and remove the else statement.

Reply:
I think yes